package.json

```diff
@@ -88,7 +89,8 @@
     "chokidar": "^2.0.4",
     "micromatch": "^3.1.10",
     "minimatch": "^3.0.4",
-    "tapable": "^1.0.0"
+    "tapable": "^1.0.0",
+    "typescript-collections": "^1.3.2"
```

Comment:
This dependency doesn't seems to be used, was it maybe included for the linked list at one point?

Reply:
you are right. removed.

Reply:
Thanks for your awesome work, I am looking forward to trying this feature out when it is merged :)